src/v2/Components/NavBar/Menus/NavBarUserMenu.tsx

```diff
@@ -65,6 +66,16 @@ export const NavBarUserMenu: React.FC = () => {
         <ReceiptIcon mr={1} aria-hidden="true" /> Order History
       </NavBarMenuItemLink>
 
+      {getENV("ENABLE_YOUR_ALERTS_PAGE") && (
```

Comment:
Nice! Might be worth documenting this `getENV` utility in our "shared configuration" doc at some point: https://github.com/artsy/force/blob/main/docs/env_configuration.md